auto_round/export/export_to_autogptq/export.py

```diff
@@ -130,8 +131,8 @@ def save_quantized_as_autogptq(output_dir, inplace=True, backend="auto_gptq:exll
         processor.save_pretrained(output_dir)
     ##check module quantized in block, this may have bug for mixed precision quantization
     quantization_config = kwargs["serialization_dict"]
-    if bool(to_quant_block_names):
-        all_blocks = to_quant_block_names
+    if bool(quant_block_list):
```

Comment:
gptq only supporst one string

Reply:
The quant_block_list will not be saved, it is just passed from the autoround class for export reference.